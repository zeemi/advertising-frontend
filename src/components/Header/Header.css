.Header {
    padding: 5px;
    box-shadow: rgba(0, 0, 0, 0.1) 0px 1px 6px, rgba(0, 0, 0, 0.1) 0px 1px 4px;
    margin-bottom: 15px;
}