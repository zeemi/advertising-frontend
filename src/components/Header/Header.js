import React from 'react';
import './Header.css'

const Header = ()=>{
  return <div className={'Header'}>
    <h3>Select zero to N Datasources</h3>
    <h3>Select zero to N Campaigns</h3>
    <div>(where zero means "All")</div>
    <h3>Hitting "Apply", filters the chart to show a timeseries for both Click and Impressions for given Datasources and Campaigns - logical AND</h3>
  </div>
};

export default Header;