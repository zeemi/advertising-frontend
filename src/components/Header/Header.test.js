import React from 'react';
import Header from './Header';
import {shallow} from 'enzyme';

describe('<Header>', function () {
  it('should render proper text', function () {
    expect(shallow(<Header/>)).toMatchSnapshot();
  });
});