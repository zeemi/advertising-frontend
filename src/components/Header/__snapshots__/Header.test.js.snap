// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header> should render proper text 1`] = `
<div
  className="Header"
>
  <h3>
    Select zero to N Datasources
  </h3>
  <h3>
    Select zero to N Campaigns
  </h3>
  <div>
    (where zero means "All")
  </div>
  <h3>
    Hitting "Apply", filters the chart to show a timeseries for both Click and Impressions for given Datasources and Campaigns - logical AND
  </h3>
</div>
`;
